### PR TITLE
Send Thalia Pay withdrawal notice emails

### LINKED ISSUE #1239: Send email with Thalia Pay payments that will be withdrawn on batch processed
### Describe the solution you'd like
When a payment batch is processed, all users that are in that batch should receive an email notifying them that we will withdraw some amount from their bank account.

### Motivation
It is required for the SEPA direct debit mandate

### PR DESCRIPTION
**Pull request by JobDoesburg**
Closes #1239 

### Summary
When a payment is getting processed, an email will be sent.

### How to test
Steps to test the changes you made:
1. Make a batch
2. Add payments to the batch
3. Process it
4. Verify that emails were sent

### CODE REVIEW

File: website/payments/templates/payments/email/tpay_withdrawal_notice_mail.txt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Dear {{ name }},
+
+In the past period, you have made use of Thalia Pay. With this email, we would like to inform you that in the upcoming days, we will withdraw an amount of € {{ total_amount|floatformat:2 }} from your bank account {{ bank_account.iban }} {% if bank_account.bic %}, {{ bank_account.bic }},{% endif %} to settle the following Thalia Pay payments:
```
Comment (se-bastiaan) [inline website/payments/templates/payments/email/tpay_withdrawal_notice_mail.txt:3]:
```suggestion
We would like to inform you that in the upcoming days, we will withdraw an amount of € {{ total_amount|floatformat:2 }} from your bank account {{ bank_account.iban }} {% if bank_account.bic %}, {{ bank_account.bic }},{% endif %} to settle the following Thalia Pay payments:
```

Comment (se-bastiaan) [reply website/payments/templates/payments/email/tpay_withdrawal_notice_mail.txt:3]: But this email needs to be send 1 month in advance doesn't it?

Comment (JobDoesburg) [reply website/payments/templates/payments/email/tpay_withdrawal_notice_mail.txt:3]: 14 days*

Comment (JobDoesburg) [reply website/payments/templates/payments/email/tpay_withdrawal_notice_mail.txt:3]: However it is a bit ambiguous as people themselves take the initiative of paying. So you could say that them clicking the 'I want to pay with Thalia Pay' button is already the so-called pre-notification.

Comment (JobDoesburg) [reply website/payments/templates/payments/email/tpay_withdrawal_notice_mail.txt:3]: I think these are the exact details we should discuss with the board

**Review by se-bastiaan — APPROVED** [review]

**Pull request merged**